plugins/inputs/aerospike/aerospike.go

```diff
@@ -161,6 +161,60 @@ func (a *Aerospike) gatherServer(hostport string, acc telegraf.Accumulator) erro
 			}
 			acc.AddFields("aerospike_namespace", nFields, nTags, time.Now())
 		}
+
+		latency, err := as.RequestNodeInfo(n, "latency:")
+		if err != nil {
+			return err
+		}
+		result, exists := latency["latency:"]
+		if !exists {
+			return errors.New("Aerospike Input Plugin : No key `latency:` in response of as.RequestNodeInfo(n, \"latency:\") from Aerospike Server")
+		}
+		points := strings.Split(result, ";")
+		i := 0
+		for i < len(points) {
+			point := points[i]
+			if strings.Compare(point, "error-no-data-yet-or-back-too-small") == 0 {
+				i++
+			} else {
+				indexOfColon := strings.Index(point, ":")
+				if indexOfColon == -1 {
+					i++
+					continue
+				}
+				histogram := point[0:indexOfColon]
+				operation, namespace := splitNamespaceAndOperation(histogram)
+				latencyKey := point[indexOfColon:]
+				latencyValue := points[i+1]
```

Comment:
Is this (`i+1`)correct?

Reply:
Ya, this is correct.
[eg., 
latency["latency:"] = error-no-data-yet-or-back-too-small;{test}-write:03:58:35-GMT,ops/sec,>1ms,>8ms,>64ms;03:58:45,0.0,0.00,0.00,0.00;error-no-data-yet-or-back-too-small;error-no-data-yet-or-back-too-small;error-no-data-yet-or-back-too-small;error-no-data-yet-or-back-too-small;error-no-data-yet-or-back-too-small;error-no-data-yet-or-back-too-small]

i -> {test}-write:03:58:35-GMT,ops/sec,>1ms,>8ms,>64ms;
i+1 -> 03:58:45,0.0,0.00,0.00,0.00;